community/modules/compute/schedmd-slurm-gcp-v6-partition/main.tf

```diff
@@ -15,6 +15,7 @@
 locals {
   non_static_ns_with_placement = [for ns in var.nodeset : ns.nodeset_name if ns.enable_placement && ns.node_count_static == 0]
   use_static                   = [for ns in concat(var.nodeset, var.nodeset_tpu) : ns.nodeset_name if ns.node_count_static > 0]
+  uses_job_duration            = length([for ns in var.nodeset : ns.dws_flex.use_job_duration if ns.dws_flex.use_job_duration]) > 0 ? true : false
```

Comment:
Having it as a list instead of bool could improve readability of error message. I recall that terraform will output values of variables used in `condition` expression.

Reply:
Yes terraform will output the values of the variables as part of the error, however in the interest of readability I think `bool` is the better option here. If its a list terraform will state ```variable is a tuple with n element``` and not actually display the contents. Whilst if its a bool it was state ```variable is true```. In our case this results in:
```
  on modules/embedded/community/modules/compute/schedmd-slurm-gcp-v6-partition/outputs.tf line 42, in output "partitions":
  42:     condition     = !local.uses_job_duration || var.exclusive
    ├────────────────
    │ local.uses_job_duration is true
    │ var.exclusive is false

`use_job_duration` can only be used in exclusive partitions
```

compared to a list:
```
  on modules/embedded/community/modules/compute/schedmd-slurm-gcp-v6-partition/outputs.tf line 42, in output "partitions":
  42:     condition     = (length(local.uses_job_duration)==0) || var.exclusive
    ├────────────────
    │ local.uses_job_duration is tuple with 1 element
    │ var.exclusive is false

`use_job_duration` can only be used in exclusive partitions
```

Reply:
`variable is a tuple with n element`, this is less descriptive than I hoped for. Resolved.